resources/docker-app/README.md

```diff
@@ -34,9 +38,9 @@ exit
 
 ### Run the Docker container in background and execute commands
 ```bash
-docker pull phopper/obs-pdf:latest
-docker run --name obs-pdf --rm -p 8080:80 -dit --cpus=0.5 phopper/obs-pdf:latest
-docker run --name obs-pdf -p 8080:80 -dit --cpus=0.5 --restart unless-stopped phopper/obs-pdf:latest
+docker pull unfoldingWord/obs-pdf:latest
+docker run --name obs-pdf --rm -p 8080:80 -dit --cpus=0.5 unfoldingWord/obs-pdf:latest
+docker run --name obs-pdf -p 8080:80 -dit --cpus=0.5 --restart unless-stopped unfoldingWord/obs-pdf:latest
```

Comment:
We don't want the `--cpus` flag on these.  If you want to reduce the CPU share that it has then I'd suggest using `--cpu-shares` and set it to a number lower than 1024, maybe 768 or 512.